UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Model/SessionConfig.cs

```diff
@@ -16,7 +16,10 @@ public class SessionConfig
 
         [JsonProperty(PropertyName = "initialPlayers")]
         public List<string> InitialPlayers { get; set; }
-
+        
+        [JsonProperty(PropertyName = "metadata")]
+        public Dictionary<string, string> Metadata { get; set; }
```

Comment:
Just worth noting how out of date the Unity GSDK is. This field was added to heartbeats a long time ago.